packages/core/src/lib/jdk/JdkInstaller.ts

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright 2019 Google Inc. All Rights Reserved.
```

Comment:
nit: wrong year for the copyright header.

```suggestion
 * Copyright 2020 Google Inc. All Rights Reserved.
```